app/assets/v2/js/pages/dashboard.js

```diff
@@ -52,6 +52,27 @@ var set_filter_header = function() {
   $("#filter").html(filter_status);
 }
 
+// TODO: Refactor function :
+// Deselect option 'any' when another filter is selected
+// Selects option 'any' when no filter is applied
+// TODO : Disable other filters when 'any' is selected
+var disableAny = function () {
```

Comment:
@eswarasai could you refactor this whenever you get to it ?  This is super messy :P

Reply:
^Aye cap'n 🖖